src/Sulu/Bundle/ContactBundle/Resources/templates/contacts.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" ?>
+<properties xmlns="http://schemas.sulu.io/template/template"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
+    <property name="formOfAddress" type="single_select" mandatory="true" colspan="3" spaceAfter="9">
+        <meta>
+            <title lang="en">Salutation</title>
+            <title lang="de">Anrede</title>
+        </meta>
+        <params>
+            <param name="default_value" value="0"/>
+            <param name="values" type="collection">
+                <param name="0" value="0">
```

Comment:
I think that is strange... Can't we just write one of them?

Reply:
We've already said that we need to implement that correctly..